images/node/Dockerfile-alpine

```diff
@@ -0,0 +1,33 @@
+###############################################################
+# DOCKER IMAGE - Alpine based Nodejs Application (Express.js) #
+###############################################################
+
+FROM alpine:latest
+
+# Install nodejs
+RUN apk add --update nodejs npm
```

Comment:
Instead of using "alpine:latest" lets just use one of these images: 

https://hub.docker.com/_/node/

Such as... 

node:18-alpine